EXCLUSION.md

```diff
@@ -1,3 +1,31 @@
-TwoFactorAuth.org Blacklist
-=================
+# TwoFactorAuth.org excluded categories and websites
 
+Below is a list of categories and websites that we've opted out from listing on TwoFactorAuth.org. If you want to make a copy (fork) of our site to list any of these sites you're welcome to do so as long as you comply with our [license](https://github.com/2factorauth/twofactorauth/blob/master/LICENSE).
+
+## Categories
+
+*   #### Pornographic sites
+
+    As the volunteers of [2factorauth](https://github.com/2factorauth) need to go through all sites to validate that the site/service provides 2FA to it's users and verify that the pull request meets our style guidelines we've chosen not to list pornographic sites.  
+    We've also chosen to do this so that we don't have to go through potentially disturbing or illegal material.
+
+*   #### Self-hosted services
+
+    The category "self-hosted services/sites" includes all types of software that's designed to be hosted by the end user (such as [ownCloud](https://owncloud.org/), [NextCloud](https://nextcloud.com/), [Mastodon](https://joinmastodon.org/), [phpBB](https://www.phpbb.com/), [XenForo](https://xenforo.com/) etc).  
+    It also includes [Wordpress.org](https://wordpress.org/) (the open source software) but not [Wordpress.com](https://wordpress.com/) (the company).
```

Comment:
As discussed in https://github.com/2factorauth/twofactorauth/pull/3550 before, i still disagree with this rule.
Bugzilla is also self-hostable. GitLab too. And you already include them.

Mastodon e.g. has one very very big instance https://mastodon.social/ that has 2FA. (and no, it's no plugin)

Reply:
... and #3476 ...

Reply:
539f4c47ff7eee2424540e28cf5adf5c621b0b01